### PR TITLE
Add css

### PR DESCRIPTION
**Pull request by hedita**

### CODE REVIEW

File: assets/css/variebles.css [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  --bold-font: 600;
+
+  --background-color: #112b3c;
+  --primary-color: #f66b0e;
+  --secondery-color: #a5c9ca;
+  --border: 1px solid #f66b0e;
+
+  --xSmall: 0.5em;
+  --small: 0.9em;
+  --Large: 3em;
```
Comment (blacksrc) [inline assets/css/variebles.css:14]: Would you please follow a same pattern across variable namings?
`--my-name` or `--MyName` or `--myName`

File: assets/css/structure.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+.wisdom-button {
+  font-weight: var(--thin-font);
+  margin: var(--small);
+  padding: var(--xSmall);
+  color: var(--secondery-color);
+}
+
+.famous-qoutes-button {
```
Comment (blacksrc) [inline assets/css/structure.css:38]: This is almost identical to the above button. The only difference is margin which I'm not sure what's the reason. First, I guess, tags are more links than buttons. So I recommend a tag instead. Second, if we have a same design for these tags/buttons. I suggest to use just one class for all.

File: index.html [hunk 3]
```diff
@@ -1,10 +1,23 @@
...
+    <link rel="stylesheet" href="./assets/css/main.css" />
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Document</title>
   </head>
-  <body></body>
+  <body>
+    <div class="section">
+      <p class="quote">
```
Comment (blacksrc) [inline index.html:12]: I suggest to use more semantic tags for quote. This article can be helpful or you can search more about that.
https://css-tricks.com/quoting-in-html-quotations-citations-and-blockquotes/

File: index.html [hunk 4]
```diff
@@ -1,10 +1,23 @@
...
+        We cannot solve our problems with the same thinking we used when we
+        created them.
+      </p>
+      <p class="author">Albert Einstein</p>
+      <div class="buttons">
+        <button class="wisdom-button">Wisdom</button>
+        <button class="famous-qoutes-button">Famous-qoutes</button>
```
Comment (blacksrc) [inline index.html:19]: These are not buttons semantically.

File: assets/css/structure.css [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+.author {
+  font-weight: var(--bold-font);
+  text-align: right;
+  margin-bottom: var(--Large);
+  color: var(--primary-color);
+}
+
+/*buttons*/
```
Comment (blacksrc) [inline assets/css/structure.css:25]: I believe this comment is redundant.

File: assets/css/structure.css (hunk 2 above)
Comment (hedita) [reply assets/css/structure.css:38]: Fixed.

File: assets/css/variebles.css (hunk 1 above)
Comment (hedita) [reply assets/css/variebles.css:14]: Done.

File: assets/css/structure.css (hunk 5 above)
Comment (hedita) [reply assets/css/structure.css:25]: I removed it.

File: assets/css/structure.css (hunk 2 above)
Comment (hedita) [reply assets/css/structure.css:38]: Fixed.

File: index.html (hunk 3 above)
Comment (hedita) [reply index.html:12]: I repleced it with q tag.

File: index.html (hunk 4 above)
Comment (hedita) [reply index.html:19]: I changed it to ul and lies tag.

**Pull request closed without merging**